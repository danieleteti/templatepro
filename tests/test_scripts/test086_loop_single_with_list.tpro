************* Object with child list of object
{{for item in dataitem.DataItemList}}
Level |{{:item.Value1}}|
{{endfor}}
