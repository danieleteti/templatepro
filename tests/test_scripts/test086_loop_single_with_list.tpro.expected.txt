************* Object with child list of object
Level |SimpleDataItem0|
Level |SimpleDataItem1|
Level |SimpleDataItem2|
